cli/src/pcluster/schemas/common_schema.py

```diff
@@ -111,3 +112,29 @@ class TagSchema(BaseSchema):
     def make_resource(self, data, **kwargs):
         """Generate resource."""
         return BaseTag(**data)
+
+
+class CookbookSchema(BaseSchema):
+    """Represent the schema of cookbook."""
+
+    chef_cookbook = fields.Str()
+    extra_chef_attributes = fields.Str()
+
+    @post_load()
+    def make_resource(self, data, **kwargs):
+        """Generate resource."""
+        return Cookbook(**data)
+
+    @validates("extra_chef_attributes")
```

Comment:
I am not sure the extra_chef_attributes only supports json format or not, we can add a TODO mark for this validation